src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="airbnb.com"
+      class="form big-search-form"
+      method="post"
+      data-qa="big"
+    >
+      <label class="field">
```

Comment:
`label` is redundant

Reply:
Why? 
I put my search image with <input> there, so it allows me to click on this image and at this moment focus on input. 
Isn't it right?